src/components/inputs/multi-select/ohri-multi-select.component.tsx

```diff
@@ -51,22 +51,22 @@ export const OHRIMultiSelect: React.FC<OHRIFormFieldProps> = ({ question, onChan
     }));
 
   const initiallySelectedQuestionItems = [];
-  questionItems.forEach(item => {
+  questionItems.forEach((item) => {
```

Comment:
Hey, why haven't you use `map` instead of forEach?

Reply:
@raghavawasthi24 Thanks for pointing this out. This PR was however for aligning the dependencies and some of the changes to files like this one was due to the linter. There's another PR that has been merged with some fixes. 

Also feel free to take a look at the codebase and suggest improvements, tickets or even PR's where you see fit.